metabase/metabase
# Use MonthPicker from mantine for dashboard parameters to enable selection of months in the future

How to verify:
- Edit dashboard
- Filter -> Date Picker -> Month and Year -> Connect to a card
- Click on the filter widget -> Click on the year -> Select a year in the future -> Select a month
- Click on the filter widget again. The month should be selected

<img width="352" alt="Screenshot 2024-12-23 at 14 58 48" src="https://github.com/user-attachments/assets/36c2780a-6f27-4cac-b24f-4486b125b2ab" />
